frontend/src/app/shared/components/op-context-menu/op-context-menu.html

```diff
@@ -7,20 +7,38 @@
     <ng-container *ngFor="let item of items">
       <li *ngIf="item.divider" class="dropdown-divider"></li>
       <li *ngIf="!item.divider">
-        <a class="menu-item {{ item.class }}"
-           tabindex="0"
-           [ngClass]="{ 'inactive': item.disabled }"
-           [attr.href]="item.href || ''"
-           [attr.aria-disabled]="item.disabled"
-           [attr.aria-label]="item.ariaLabel || item.linkText"
-           (accessibleClick)="handleClick(item, $event)"
-           [accessibleClickStopEvent]="false">
-          <op-icon *ngIf="item.icon" icon-classes="icon-action-menu {{ item.icon }}"> </op-icon>
+        <a
+          *ngIf="item.href"
+          class="menu-item"
+          tabindex="0"
```

Comment:
Should be a button, and we can probably remove the tabindex, since this can receive focus anyway

Reply:
There's an ngIf to show a button if there's no link, and a link if there's one. This ensures the native click handling on the link works so I'd leave it as is, but yes we can remove the tabindex